### PR TITLE
Created markdown for C# EndsWith Term Entry

### LINKED ISSUE #1837: [Term Entry] C# Strings `.EndsWith()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the `String.EndsWith()` method.

The entry should go in a new file in docs/content/c-sharp/concepts/strings/terms/endswith/endswith.md

The entry should

- Have a syntax section illustrating the method with a ` ```pseudo ` code block.
- An example showing the method in use with a ` ```cs ` code block.
- A runnable codebyte example in a ` ```codebyte/csharp ` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ldself**
### Description

Adding a Term Entry for the C# String.EndsWith() method.  I was working through the Learn Git & Github course on Codecademy and thought I would try my first pull request. (Closes #1837)

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/strings/terms/endswith/endswith.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+Tags:
+  - 'Strings'
+  - 'Characters;
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.EndsWith()`** method determines whether the end of the String instance matches the specified character or string.  Additional parameters may be supplied to define the criteria for a match.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/endswith/endswith.md:16]:
```suggestion
The **`.EndsWith()`** method determines whether the end of the `String` instance matches the specified character or string.  Additional parameters may be supplied to define the criteria for a match.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ldself I did a first review and only had one comment.

File: content/c-sharp/concepts/strings/terms/endswith/endswith.md (hunk 1 above)
Comment (ldself) [reply content/c-sharp/concepts/strings/terms/endswith/endswith.md]: @SSwiniarski I made the change that you suggested and pushed the updated file.  Thanks.

**Review by SSwiniarski — APPROVED** [review]
@ldself looks good for a second review.

File: content/c-sharp/concepts/strings/terms/endswith/endswith.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
+---
+Title: '.EndsWith()'
+Description: 'Determines if the end of the string instance matches the specified string and returns a boolean.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Characters;
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/endswith/endswith.md:9]:
```suggestion
  - 'Characters'
```

**Review by SSwiniarski — COMMENTED** [review]
@ldself One more thing.

**Comment by ldself** [comment]
@SSwiniarski I'm having trouble getting 'yarn format:verify' to work on my local machine.  I put a question on the discussion board about it and I wanted to see if you could take a look at it.  It basically, can't find the @codecademy/prettier-config module and I'm not sure how to resolve it.  Thanks.

https://github.com/Codecademy/docs/discussions/1864

**Comment by SSwiniarski** [comment]
@KTom101, @yangc95, @caupolicandiaz, any suggestions for @ldself's problem?

**Comment by ldself** [comment]
@SSwiniarski I've made the additional suggested change and fixed the formatting. Thanks to @KTom101 for providing a solution to my earlier problem.  The updated branch has been pushed.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Idself I've completed my review. Please update, and tag me when you've finished making changes. Thanks

**Comment by ldself** [comment]
Thanks for the feedback @caupolicandiaz.  Changes have been pushed.

**Comment by SSwiniarski** [comment]
@ldself congrats on the new entry.  Live link is here: https://www.codecademy.com/resources/docs/c-sharp/strings/endswith

You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

Note: Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

**Pull request merged**